com.obsproject.Studio.Plugin.Kinect.metainfo.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.Kinect</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>obs-kinect</name>
+  <summary>Use Xbox Kinect and remove background</summary>
```

Comment:
Not a native speaker, but I think the more appropriate form would be `Use Xbox Kinect to remove background`, isn't it?

Reply:
Reading it again, it does indeed sound off to me. I'm just trying emphasize the fact that it allows the Kinect to be used as a source and has the optional ability to remove/change the background. How does `Add Xbox Kinect as a source and change the video background of it` sound to you? I'm fine with whatever either way. So if you're set on it, I'll keep it as `Use Xbox Kinect to remove background`